### PR TITLE
CLM for PRs/FRs

### PR DESCRIPTION
**Pull request by planteater**
1. Clarify distributed tracing requirement for CLM
2. Add support email address to main help article. I know this may be a bit unusual, but this is the page our MS Teams app links to and Microsoft requires that a means of contact be displayed there.
3. Add copy/image about new support for code-level metrics in PR/FR diffs.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

### CODE REVIEW

File: src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx [hunk 1]
```diff
@@ -269,11 +271,17 @@ Code-level metrics give you detailed insight into how your code is performing at
   src={codeLevelMetrics}
 />
 
-<figcaption>Code-level metrics are displayed above each instrumented method in your Python/Ruby project.</figcaption>
+Your code reviews can now also be more data driven by leveraging golden metrics during the review process. Code-level metrics are displayed in the diffs for both pull requests and feedback requests, allowing you to make sure that performance issues get addressed as part of the changes. 
+
+<img
+  title="A diff with golden metrics."
+  alt="A screenshot of a deff with golden metrics."
```
Comment (paperclypse) [inline src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx:278]:
```suggestion
  alt="A screenshot of a diff with golden metrics."
```

File: src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx [hunk 2]
```diff
@@ -269,11 +271,17 @@ Code-level metrics give you detailed insight into how your code is performing at
...
+  src={codeLevelDiffs}
+/>
 
 ### Requirements [#clm-requirements]
 
-* [Distributed tracing](/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing) must be enabled.
+* [Distributed tracing](/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing) must be enabled. Note that is is on by default in recent versions of all agents, so you only need to worry about this if you're disabled distributed tracing.
```
Comment (paperclypse) [inline src/content/docs/codestream/how-use-codestream/performance-monitoring.mdx:284]:
```suggestion
* [Distributed tracing](/docs/distributed-tracing/enable-configure/language-agents-enable-distributed-tracing) must be enabled. Distributed tracing is on by default in recent versions of all agents, so you only need to worry about this if you're disabled distributed tracing.
```

**Review by paperclypse — APPROVED** [review]
Looks good. I've made some minor changes.

**Review by akristen — APPROVED** [review]
Thanks @planteater and @paperclypse. :)

**Pull request merged**